db/migrate/20230503133357_create_likes.rb

```diff
@@ -0,0 +1,10 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :user, null: false, foreign_key: true
+      t.references :post, null: false, foreign_key: true
+
```

Comment:
- Great work on your migration files, however, I would recommend you use `author_id` when referencing the `user` and `post_id` when referencing `post` in your likes table as it is in the ERD diagram 😉